src/main/kotlin/sliv/tool/main/MainController.kt

```diff
@@ -9,16 +10,17 @@ import java.time.LocalDateTime
 import java.time.ZoneId
 
 class MainController : Controller() {
-    private val sceneController: SceneController by inject()
-    private val sceneFacade = SceneFacade(sceneController)
+    private val catalogueController: CatalogueController by inject()
 
     //TODO: temporary solution to fill scene with data
     fun importTestData() {
         val layer = ProjectLayer(LayerKind.KEYPOINT, "ORB")
         val path = Paths.get("test/data/room.jpg")
-        val timestamp = LocalDateTime.now().atZone(ZoneId.of("UTC+03:00")).toInstant().toEpochMilli()
+        val currentTimestamp = {
+            LocalDateTime.now().atZone(ZoneId.of("UTC+03:00")).toInstant().toEpochMilli()
+        }
         val landmarks = listOf(LandmarkFile(layer, Paths.get("test/data/points.csv"), emptyList()))
-        val frames = (1..30000).map { ProjectFrame(timestamp, path, landmarks) }
-        sceneFacade.visualize(listOf(layer), frames)
+        val frames = (1..30000).map { ProjectFrame(currentTimestamp(), path, landmarks) }
+        catalogueController.displayFrames(frames)
```

Comment:
It would be nice if all the frames will be selected by default. So a user will see frames instantly when project was imported.